addon-test-support/components/ember-ace/autocomplete.js

```diff
@@ -7,7 +7,8 @@ import { text, collection, is, isVisible } from 'ember-cli-page-object';
  * and metadata on the right.
  */
 export const suggestion = {
-  selected: is('.ace_selected'),
+  // selected: is('.ace_selected'),
+  selected: is('.ace_active-line'),
```

Comment:
The test is failing because `.ace_active-line` isn't a modifier that gets added to the currently selected `.ace_line`. It's a _completely separate element in a completely separate part of the DOM_ that happens to have the same X/Y coordinates. I can't think of a good way to express that as a jQuery selector like `is('.ace_selected')`.